<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XFile Manager</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/Res/icons/logo.png</normalon>
   </iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgba(38, 34, 39, 23); /* Slightly transparent for a blur-like effect */
color: rgb(255, 255, 255); /* White text color */</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QScrollBar:vertical {
    border: 2px solid #cccccc; /* Border color */
    background: #f1f1f1; /* Background color */
    width: 10px; /* Width of the vertical scrollbar */
    margin: 22px 0 22px 0; /* Margins for the vertical scrollbar */
}

QScrollBar::handle:vertical {
    background: #aaaaaa; /* Color of the scrollbar handle */
    min-height: 20px; /* Minimum height for the handle */
}

QScrollBar::add-line:vertical {
    background: none; /* Hides the add button */
    height: 0px; /* Height for the add button */
}

QScrollBar::sub-line:vertical {
    background: none; /* Hides the sub button */
    height: 0px; /* Height for the sub button */
}

QScrollBar:horizontal {
    border: 2px solid #cccccc; /* Border color */
    background: #f1f1f1; /* Background color */
    height: 10px; /* Height of the horizontal scrollbar */
    margin: 0px 22px 0 22px; /* Margins for the horizontal scrollbar */
}

QScrollBar::handle:horizontal {
    background: #aaaaaa; /* Color of the scrollbar handle */
    min-width: 20px; /* Minimum width for the handle */
}

QScrollBar::add-line:horizontal {
    background: none; /* Hides the add button */
    width: 0px; /* Width for the add button */
}

QScrollBar::sub-line:horizontal {
    background: none; /* Hides the sub button */
    width: 0px; /* Width for the sub button */
}
</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SizeConstraint::SetFixedSize</enum>
        </property>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>500</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
background-color: rgb(0, 0, 0);/* Default background color */
 color: rgb(122, 122, 122);            /* Default text color */
border:none;
outline:none;
border-radius:6px;
text-align:left;
padding-left:4px;
padding-top:10px;
padding-bottom:10px;
}
QPushButton:hover {
background-color: rgb(65, 65, 65) /* Default background color */
}
QPushButton:pressed,
QPushButton:focus {
    background-color: #252126; /* Background color when selected */
    color: #FFFFFF;            /* Text color when selected */
}
outline:none;
border:none;
</string>
          </property>
          <property name="title">
           <string>Favourites</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <property name="spacing">
            <number>4</number>
           </property>
           <property name="leftMargin">
            <number>6</number>
           </property>
           <property name="topMargin">
            <number>6</number>
           </property>
           <property name="rightMargin">
            <number>6</number>
           </property>
           <property name="bottomMargin">
            <number>6</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>2</number>
             </property>
             <item>
              <widget class="QPushButton" name="home">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LayoutDirection::LeftToRight</enum>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Desktop</string>
               </property>
               <property name="icon">
                <iconset resource="XFile_Manager_Resources.qrc">
                 <normaloff>:/Res/icons/Desktop.svg</normaloff>
                 <normalon>:/Res/icons/Desktop.svg</normalon>:/Res/icons/Desktop.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>26</width>
                 <height>26</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Downlaod_btn">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Downloads</string>
               </property>
               <property name="icon">
                <iconset resource="XFile_Manager_Resources.qrc">
                 <normaloff>:/Res/icons/Downlaod.svg</normaloff>:/Res/icons/Downlaod.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>26</width>
                 <height>26</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Doc_btn">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Documents</string>
               </property>
               <property name="icon">
                <iconset resource="XFile_Manager_Resources.qrc">
                 <normaloff>:/Res/icons/doc.svg</normaloff>
                 <normalon>:/Res/icons/doc.svg</normalon>:/Res/icons/doc.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>26</width>
                 <height>26</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="images_btn">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Pictures</string>
               </property>
               <property name="icon">
                <iconset resource="XFile_Manager_Resources.qrc">
                 <normaloff>:/Res/icons/images.svg</normaloff>:/Res/icons/images.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>26</width>
                 <height>26</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Music_btn">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Music</string>
               </property>
               <property name="icon">
                <iconset resource="XFile_Manager_Resources.qrc">
                 <normaloff>:/Res/icons/music.svg</normaloff>:/Res/icons/music.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>26</width>
                 <height>26</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Video_btn">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Videos</string>
               </property>
               <property name="icon">
                <iconset resource="XFile_Manager_Resources.qrc">
                 <normaloff>:/Res/icons/video.svg</normaloff>:/Res/icons/video.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>26</width>
                 <height>26</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>160</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
background-color: rgb(0, 0, 0);/* Default background color */
 color: rgb(122, 122, 122);            /* Default text color */
border:none;
outline:none;
border-radius:6px;
text-align:left;
padding-left:4px;
padding-top:10px;
padding-bottom:10px;
}
QPushButton:hover {
background-color: rgb(65, 65, 65) /* Default background color */
}
QPushButton:pressed,
QPushButton:focus {
    background-color: #252126; /* Background color when selected */
    color: #FFFFFF;            /* Text color when selected */
}
outline:none;
border:none;
</string>
          </property>
          <property name="title">
           <string>System</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="spacing">
            <number>4</number>
           </property>
           <property name="leftMargin">
            <number>6</number>
           </property>
           <property name="topMargin">
            <number>6</number>
           </property>
           <property name="rightMargin">
            <number>6</number>
           </property>
           <property name="bottomMargin">
            <number>6</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="sizeConstraint">
              <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
             </property>
             <item>
              <widget class="QPushButton" name="home_main_btn">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Home</string>
               </property>
               <property name="icon">
                <iconset resource="XFile_Manager_Resources.qrc">
                 <normaloff>:/Res/icons/Home.svg</normaloff>:/Res/icons/Home.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>26</width>
                 <height>26</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="system_vol_btn">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>System Volume</string>
               </property>
               <property name="icon">
                <iconset>
                 <normalon>:/Res/icons/system_volume.svg</normalon>
                </iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>26</width>
                 <height>26</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="C_drive">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>C Drive</string>
               </property>
               <property name="icon">
                <iconset resource="XFile_Manager_Resources.qrc">
                 <normaloff>:/Res/icons/drive.svg</normaloff>:/Res/icons/drive.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>26</width>
                 <height>26</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SizeConstraint::SetMinimumSize</enum>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="sizeConstraint">
           <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QPushButton" name="back_btn">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normalon>:/Res/icons/left.svg</normalon>
             </iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>26</width>
              <height>26</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="forward_btn">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normalon>:/Res/icons/right.svg</normalon>
             </iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>26</width>
              <height>26</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="current_path">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>XFile Manager</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="copy_path_btn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: #252126;</string>
            </property>
            <property name="text">
             <string>Copy Path</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="info_svg">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: #252126;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normalon>:/Res/icons/info.svg</normalon>
             </iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListView" name="listView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QListView {
            background-color: #252126;
            color: rgb(255, 255, 255);
        }
        QListView::item {
            padding: 10px; /* Adjust padding for items */
        }
        QListView::item:hover {
            background-color: #3b2e45; /* Change to your desired hover color */
            color: rgb(255, 255, 255); /* Text color on hover */
        }</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeView" name="treeView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">
        QTreeView {
            background-color: #252126;
            color: rgb(255, 255, 255);
        }
        QTreeView::item {
            padding: 10px; /* Adjust padding for items */
        }
        QTreeView::item:hover {
            background-color: #3b2e45; /* Change to your desired hover color */
            color: rgb(255, 255, 255); /* Text color on hover */
        }</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QListWidget" name="listWidget">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>120</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">
    QPushButton {
        background-color: #252126;
		text-align:left;
		height:100px;
		
    }

    QPushButton:hover {
        background-color: rgb(0, 0, 0); /* Darker green on hover */
    }

    QPushButton:pressed {
        background-color: rgb(62, 62, 62); /* Even darker green when pressed */
        padding: 14px 31px; /* Slightly smaller padding when pressed */
    }
</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QScrollArea" name="scrollArea_system_info">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>120</height>
             </size>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>134</width>
               <height>110</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_5"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>831</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="XFile_Manager_Resources.qrc"/>
 </resources>
 <connections/>
</ui>
